api/document/tests/end_to_end_test.py

```diff
@@ -0,0 +1,55 @@
+from io import BytesIO
+from pathlib import Path
+
+import pytest
+from django.core.management import call_command
+from model_mommy import mommy
+
+from document.models import DocNode
+from document.parsers import AkomaNtosoParser
+from document.renderers import AkomaNtosoRenderer
+from document.serializers.doc_cursor import DocCursorSerializer
+from document.tree import DocCursor
+from reqs.models import Policy
+
+MY_DIR = Path(__file__).parent.resolve()
+
+EXAMPLE_DOCS_DIR = MY_DIR / '..' / '..' / 'example_docs'
+
+
+@pytest.mark.django_db
+def test_akn_works():
+    # Phase 1: Import the document from XML, serialize it, and
+    # render it to AKN.
+    policy = mommy.make(Policy, omb_policy_id='M-16-19')
+    call_command('import_xml_doc', str(EXAMPLE_DOCS_DIR / 'm_16_19_1.xml'),
+                 'M-16-19')
+    docnode = DocNode.objects.filter(policy=policy, depth=1).first()
+    cursor = DocCursor.load_from_model(docnode)
```

Comment:
Should we use that prefetched version of the queryset to speed this up?

Reply:
Yeah! At the time I wrote this, that PR wasn't merged yet, but now that it is, I will use it.